src/styles.css

```diff
@@ -258,4 +264,43 @@ li {
     width: 1px;
     height: 1px;
     overflow: hidden;
+}
+
+@media print {
```

Comment:
This a great to know for formatting future print need!

Reply:
+1 - great idea!